test/integration_config.json

```diff
@@ -6,6 +6,13 @@
     "serviceport": 80,
     "repo": "gambtho/go_echo"
   },
+  {
+    "language": "go",
+    "version": "1.20",
+    "port": "8080",
+    "serviceport": 8080,
+    "repo": "davidgamero/go-echo-no-mod"
```

Comment:
💡 fyi test had your alias - is this meant to be that way or placeholder for user alias et. al. Thanks heaps.

Reply:
🧙  but then I see other example with other user alias so all good, jsut fyi.

Reply:
yep the integration tests pull use those repos to run draft against for validating it works on each language